### PR TITLE
Translation 3 files to Ko 

### PR DESCRIPTION
**Pull request by seohyun0120**
en:
- [Babel with TypeScript.md](https://github.com/microsoft/TypeScript-Website/blob/v2/packages/documentation/copy/en/tutorials/Babel%20with%20TypeScript.md)
- [React.md](https://github.com/microsoft/TypeScript-Website/blob/v2/packages/documentation/copy/en/tutorials/React.md)
- [TypeScript Tooling in 5 minutes.md](https://github.com/microsoft/TypeScript-Website/blob/v2/packages/documentation/copy/en/tutorials/TypeScript%20Tooling%20in%205%20minutes.md)

번역 리뷰 요청합니다 : @yeonjuan @guyeol @dvlprsh

ref: #910

### CODE REVIEW

File: packages/documentation/copy/ko/tutorials/TypeScript Tooling in 5 minutes.md [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+TypeScript로 간단한 웹 애플리케이션을 만들어봅시다.
+
+## TypeScript 설치하기
+
+프로젝트에서 TypeScript를 사용할 수 있는 두 가지 주요 방법이 있습니다:
+
+- npm 사용하기(Node.js 패키지 매니저)
+- Visual Studio TypeScript 플러그인 설치하기
+
+Visual Studio 2017과 Visual Studio 2015 Update 3에는 TypeScript가 기본적으로 포함되어있습니다.
```
Comment (dvlprsh) [inline packages/documentation/copy/ko/tutorials/TypeScript Tooling in 5 minutes.md]:
```suggestion
Visual Studio 2017과 Visual Studio 2015 Update 3에는 TypeScript가 기본적으로 포함되어 있습니다.
```

File: packages/documentation/copy/ko/tutorials/TypeScript Tooling in 5 minutes.md [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+만약 Visual Studio로 TypeScript를 다운받지 않았다면, [여기서 다운로드받으세요](/download).
+
+npm 사용자들:
```
Comment (dvlprsh) [inline packages/documentation/copy/ko/tutorials/TypeScript Tooling in 5 minutes.md]:
```suggestion
npm 사용자의 경우:
```

File: packages/documentation/copy/ko/tutorials/TypeScript Tooling in 5 minutes.md [hunk 3]
```diff
@@ -0,0 +1,187 @@
...
+
+## TypeScript 설치하기
+
+프로젝트에서 TypeScript를 사용할 수 있는 두 가지 주요 방법이 있습니다:
+
+- npm 사용하기(Node.js 패키지 매니저)
+- Visual Studio TypeScript 플러그인 설치하기
+
+Visual Studio 2017과 Visual Studio 2015 Update 3에는 TypeScript가 기본적으로 포함되어있습니다.
+만약 Visual Studio로 TypeScript를 다운받지 않았다면, [여기서 다운로드받으세요](/download).
```
Comment (dvlprsh) [inline packages/documentation/copy/ko/tutorials/TypeScript Tooling in 5 minutes.md]: [제안]
```suggestion
만약 Visual Studio로 TypeScript를 설치하지 않았다면, [여기서 다운로드하세요](/download).
```

File: packages/documentation/copy/ko/tutorials/Babel with TypeScript.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+
+JavaScript 코드 베이스에서 TypeScript로 포팅되었을 수 있는 기존 빌드 인프라 구조를 가진 프로젝트의 일반적인 패턴입니다.
+
+이 기술은, Babel의 [preset-typescript](https://babeljs.io/docs/en/babel-preset-typescript)을 사용하여 JS 파일을 생성한 후, TypeScript를 사용하여 타입 검사 및 `.d.ts` 파일을 생성하는 복합 접근 방식입니다.
+
+Babel은 TypeScript를 지원하기 때문에, 기존 빌드 파이프라인으로 작업할 수 있고 Babel이 코드 타입을 검사하지 않기 때문에 JS 출력 시간이 더 빨라질 수 있습니다.
+
+#### 타입 검사와 d.ts 파일 생성 (Type Checking and d.ts file generation)
+
+Babel 사용의 단점은 TS를 JS로 전환하는 동안 타입 검사를 할 수 없다는 점입니다. 즉, 에디터에서 타입 오류를 잡지 못하고 상용 코드에 포함될 수도 있단 뜻입니다.
```
Comment (dvlprsh) [inline packages/documentation/copy/ko/tutorials/Babel with TypeScript.md]: [제안]
```suggestion
Babel 사용의 단점은 TS를 JS로 전환하는 동안 타입 검사를 할 수 없다는 점입니다. 즉, 에디터에서 잡지 못한 타입 오류가 프로덕션 코드에 포함될 수도 있다는 뜻입니다.
```

Comment (seohyun0120) [reply packages/documentation/copy/ko/tutorials/Babel with TypeScript.md]: 훨씬 더 깔끔해진 것 같아요 :)

**Review by dvlprsh — APPROVED** [review]
LGTM 👍

File: packages/documentation/copy/ko/tutorials/Babel with TypeScript.md [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+모던 JavaScript 프로젝트를 만들 때, TypeScript에서 JavaScript 파일로 변환하는 올바른 방법에 대해 고민할 수 있습니다.
+
+많은 경우 그 대답은 프로젝트에 따라 _"~에 달려있다"_ 또는 _"누군가 여러분 대신 결정했을지도 모른다`_ 가 될 것입니다. 만약 [tsdx](https://www.npmjs.com/package/tsdx), [Angular](https://angular.io/), [NestJS](https://nestjs.com/)와 같은 기존 프레임워크 또는 [Getting Started](/docs/home)에 언급된 프레임워크를 사용하여 프로젝트를 만들고 있다면 결정은 여러분 손에 달려있습니다.
+
+하지만, 사용할만한 직관적인 방법은 다음과 같습니다:
+
+- 빌드 출력 결과와 소스 입력 파일이 거의 비슷한가요? `tsc`를 사용하세요.
+- 여러 잠재적인 결과물을 내는 빌드 파이프라인이 필요하신가요? `babel`로 트랜스파일링을 하고, `tsc`로 타입을 검사하세요.
+
+## 트랜스파일링은 Babel, 타입은 `tsc` (Babel for transpiling, `tsc` for types)
```
Comment (bumkeyy) [inline packages/documentation/copy/ko/tutorials/Babel with TypeScript.md]:
```suggestion
## 트랜스파일링은 Babel, 타입은 `tsc`
```

File: packages/documentation/copy/ko/tutorials/Babel with TypeScript.md [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+
+JavaScript 코드 베이스에서 TypeScript로 포팅되었을 수 있는 기존 빌드 인프라 구조를 가진 프로젝트의 일반적인 패턴입니다.
+
+이 기술은, Babel의 [preset-typescript](https://babeljs.io/docs/en/babel-preset-typescript)을 사용하여 JS 파일을 생성한 후, TypeScript를 사용하여 타입 검사 및 `.d.ts` 파일을 생성하는 복합 접근 방식입니다.
+
+Babel은 TypeScript를 지원하기 때문에, 기존 빌드 파이프라인으로 작업할 수 있고 Babel이 코드 타입을 검사하지 않기 때문에 JS 출력 시간이 더 빨라질 수 있습니다.
+
+#### 타입 검사와 d.ts 파일 생성 (Type Checking and d.ts file generation)
```
Comment (bumkeyy) [inline packages/documentation/copy/ko/tutorials/Babel with TypeScript.md]:
```suggestion
#### 타입 검사와 d.ts 파일 생성
```

File: packages/documentation/copy/ko/tutorials/Babel with TypeScript.md (hunk 5 above)
Comment (bumkeyy) [reply packages/documentation/copy/ko/tutorials/Babel with TypeScript.md]: 현재 레포에서는 다른 국가들의 번역 규칙에 따라 원문을 삽입하지 않기로 했습니다 :)

**Comment by bumkeyy** [comment]
고생하셨습니다 👍

**Comment by bumkeyy** [comment]
LGTM

**Pull request merged**